src/dombeck_lab_to_nwb/azcorra2023/photometry_utils/add_fiber_photometry.py

```diff
@@ -44,6 +44,7 @@ def add_fiber_photometry_series(
     rate: float,
     fiber_photometry_series_name: str,
     table_region_ind: int = 0,
+    unit: str = "F",
```

Comment:
"F" stand for fluorescence? Shouldn't we use the SI notation here? Usually the fluorescence does not have a u.m

Reply:
@alessandratrapani 
I copied it from the extension example: https://github.com/catalystneuro/ndx-fiber-photometry/blob/94d80c2996cf3c71bd7e6f2b2c83f9b3fb9906ff/src/pynwb/tests/integration/test_fiber_photometry.py#L203
should it be corrected to "n.a." ?

Reply:
yes it should be n.a.

Reply:
Okay I'll change this in a follow-up